### DISCUSSION TITLE
Metadata of Source-Target Mapping

### DISCUSSION THREAD

**Discussion by rk186009** [open-metadata/OpenMetadata#5433] [Q&A] [answered]
This looks to be a great project! My compliments for endeavoring to fill the gap in this area.

I could not find any entity that helps capture the metadata related to source-to-target mapping related to data pipelines. I am new to this project, hence, just checking if I am missing things here. This metadata is important in lineage and impact analysis, apart from other uses.

**Comment by harshach** [comment, marked as answer]
@rk186009 Thanks for the kind words. Yes we do have Lineage APIs and schemas. Please check the lineage schema here https://github.com/open-metadata/OpenMetadata/blob/main/catalog-rest-service/src/main/resources/json/schema/type/entityLineage.json and addLineage request https://github.com/open-metadata/OpenMetadata/blob/main/catalog-rest-service/src/main/resources/json/schema/api/lineage/addLineage.json . Please let us know if you have any questions

**Reply by rk186009** [reply]
That answers the questions. Thanks @harshach !
